kernel/write_uint63.ml

```diff
@@ -31,8 +31,8 @@ let ml_file_copy input output =
 
 let write_uint63 () =
   ml_file_copy
-    (if max_int = 1073741823 (* 32-bits *) then "uint63_x86.ml"
-     else (* 64 bits *) "uint63_amd64.ml")
+    (if max_int = 1073741823 (* 32-bits *) then "uint63_i386_31.ml"
+     else (* 64 bits *) "uint63_amd64_63.ml")
     "uint63.ml"
 
 let () = write_uint63 ()
```

Comment:
I guess this is kept for make
Alternatively we could dispatch in configure, ie
- configure writes `INT_SIZE` to config/Makefile
- Makefile.build copies `uint63_$(INT_SIZE)`

Reply:
Indeed, this is kept for make, as well as the copy program.